wafer/talks/admin.py

```diff
@@ -70,7 +71,18 @@ class TrackAdmin(VersionAdmin, admin.ModelAdmin):
     readonly_fields = ('css_class',)
 
 
+class ReviewScoreInline(admin.TabularInline):
+    model = Score
+
+
+class ReviewAdmin(admin.ModelAdmin):
+    list_display = ('talk', 'reviewer', 'total_score')
+    inlines = (ReviewScoreInline,)
+
+
```

Comment:
It would be useful to expose the talk and review update times from reversion somewhere, so we can spot cases where the talk has been updated after a review has been submitted.

Reply:
Talks aren't currently "reversion"ed. Should we?

Reply:
Revisions are created when talks are edited  - see talks/views.py

Reviews should also be handled by reversion.

Reply:
FWIW as a content team leader on a conference which runs on Wafer, I think something like this could work:

1. If a talk is modified and has not been rated, allow it to be modified. For our purposes, it wouldn't even have to be revisioned.
2. If a talk has been rated but is not yet published, modifying it should be versioned (extra points if versioning means a diff is available). The talk's ratings are marked as "stale", content team members must confirm them or modify them.
3. If a talk has been accepted, it should _not_ be modifiable anymore except by the content team. Authors that want to modify accepted talks are encouraged to mail content@ indicating the changes.

Reply:
2 (and 1) should happen automatically. Diffs for talks are available from the admin interface. Depending on the user interface we want for reviews, we may need to hook up diff views outside the admin interface, as we've done for pages (perhaps with more than just the "previous version" comparison that the page editing UI supports).

Once the talk status is changed to "Pending", "Accepted" or "Rejected", it is no longer editable by the authors, so that should take care of 3, although we may want more fine-grained control of talk editing permissions than the current "admin access only" model.